### ISSUE TITLE
Add tips when calling commands

### ISSUE THREAD

**Issue by SGA-A** [SGA-A/c2c#44]
**Is your feature request related to a problem? Please describe.**
Some players may often form bad practises in the use of the bot. 
Adding tips when calling commands resolves this by informing users of QoL features.

**Describe the solution you'd like**
- Be able to toggle tips for a user on and off
- The tips the user receives varies based on their situation
- Do not barge the user with tips
- If there is something that was done in a command that could have been done in a better way, inform a user in 

Tips by default should be enabled.
Tips should be concise, only explaining what to do better, and how to do it.
Tips should be ephemeral.

**Additional context**
See command below. This was sent because the daily command was called.
![image](https://github.com/SGA-A/c2c/assets/73552283/622fc4ae-047c-429c-b18a-716fd958b752)

**Comment by SGA-A** [comment]
Nearly all the criteria we've set for this feature has been implemented. 
All that's left is to work on giving the user the ability to toggle tips on or off.
This is probably the hardest part because we're not entirely sure where to provide this.

We've got a few options: 
- A settings slash command to manage your own preferences (using a dropdown to select it)
- A button that displays on the tip when it is displayed
- Somewhere else (we're haven't figured out where yet)

Also, another feature I'd like to add is to increase the user's multiplier if they have tips enabled. 
This is also relevant in [this issue.](https://github.com/SGA-A/c2c/issues/35)
